docs/component-readme-examples/react-native-readme-example.md

```diff
@@ -0,0 +1,95 @@
+# ComponentName
+
+ComponentName is used to render standardized elements within an interface.
+
```

Comment:
Maybe add Developer Note here

Reply:
I was expecting developer notes to be an exception to the general documentation templates. I think we just want to make sure the standard information is covered. Do you think there will be many developer notes?